### PR TITLE
fix: npm audit issues

### PR DESCRIPTION
**Pull request by L2jLiga**
**Summary**
Fixed npm audit issues by updating packages

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [x] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No
- [x] Not sure

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [x] DO NOT include files inside `lib` directory.

### CODE REVIEW

File: build/build.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const rollup = require('rollup')
 const buble = require('rollup-plugin-buble')
 const commonjs = require('rollup-plugin-commonjs')
 const nodeResolve = require('rollup-plugin-node-resolve')
-const uglify = require('rollup-plugin-uglify')
+const { uglify } = require('rollup-plugin-uglify')
```
Comment (timaschew) [inline build/build.js]: This change is not related to npm audit

Comment (L2jLiga) [reply build/build.js]: Last version of `rollup-plugin-uglify` uses named exports, another option here is to use 
`const uglify = require('rollup-plugin-uglify').uglify`

Comment (timaschew) [reply build/build.js]: Oh my fault, I though this change was related to whitespaces only. Sorry!

**Comment by L2jLiga** [comment]
@felangel, @timaschew are you going to release new version?

**Comment by felangel** [comment]
@timaschew whats the process for releasing?

**Comment by anikethsaha** [comment]
@felangel any update on this release ?

**Comment by anikethsaha** [comment]
We need this for `master` and then release it

**Comment by timaschew** [comment]
>@timaschew whats the process for releasing?

1. merge to master
2. run build/release.sh (you need permission on npm)

Let's add this to the https://github.com/docsifyjs/docsify/blob/develop/CONTRIBUTING.md

**Comment by decompil3d** [comment]
Hi folks -- is there a plan to get this released soon? Dependent packages are getting GitHub security alerts because of the old version of `marked` used in the latest published version. Thanks in advance for your help!

**Comment by anikethsaha** [comment]
Sorry for the delay, It will be released soon. need some time for this. 

cc @felangel @timaschew

**Comment by anikethsaha** [comment]
@decompil3d Done 🎉  https://www.npmjs.com/package/docsify/v/4.10.0

**Comment by decompil3d** [comment]
Thank you @anikethsaha 👏
